onnx/defs/reduction/utils.cc

```diff
@@ -61,17 +61,19 @@ to `False` instead of `True`.)DOC";
     if (axes_input) {
       schema.Attr(
           "noop_with_empty_axes",
-          "Defines behavior if 'axes' is empty. Default behavior with 'false' is to reduce all axes. "
-          "When axes is empty and this attribute is set to true, input tensor will not be reduced,"
-          "and the output tensor would be equivalent to input tensor.",
+          "Defines behavior when 'axes' is explicitly provided but empty. "
+          "If false (default), reduction happens over all axes. "
+          "If true, input tensor is returned unchanged (acts as Identity op).",
```

Comment:
Change this line to say "If true, no reduction is applied, but other operations may be performed. For example, ReduceSumSquare acts as a vanilla Square."